src/Microsoft.Win32.Registry/src/PinvokeAnalyzerExceptionList.analyzerdata

```diff
@@ -0,0 +1,21 @@
+advapi32.dll!RegEnumValueW
```

Comment:
This assembly needs to not build for UAP at all, when https://github.com/dotnet/corefx/issues/15966  is fixed this file can be deleted.